### ISSUE TITLE
Error opening a box

### ISSUE THREAD

**Issue by aldairbotherotaku** [PippleCultist/HoloCureMultiplayerMod#2]
When the client player hasn't the stage or item generated from a box the games crash
Also sometimes client can get abilities without beeing count as

**Comment by PippleCultist** [comment]
Can you clarify on what your issues are? I'm not sure what you mean by `When the client player hasn't the stage or item generated from a box` or `client can get abilities without beeing count as`. Also, do you have the error messages when the game crashed?

**Comment by seanvanpelt2** [comment]
This just happened to my friend and I. I was hosting the game and he was the client player. When he went to get a the Holozon box the game crashed. The OP is saying that when the Client Player doesn't have an item unlocked that was generated by the Holozon box, the game will crash.

**Comment by aldairbotherotaku** [comment]
> Can you clarify on what your issues are? I'm not sure what you mean by `When the client player hasn't the stage or item generated from a box` or `client can get abilities without beeing count as`. Also, do you have the error messages when the game crashed?

I would share the logs, but since i wan't hosting it i just took a quick look at them, i don't have them

**Comment by kd8lvt** [comment]
Got a log for y'all! :)
```
___________________________________________
############################################################################################
ERROR in
action number 1
of Draw Event
for object obj_PlayerManager:

Unable to get variable moneyGain from object 000001CEF362B000
0> CallbackManagerMod!_AurieFrameworkInit+0x1C632
1> HoloCure+0x21576F0
2> HoloCure+0x1C1856E
3> HoloCure+0x2158EE9
4> YYToolkit!ModuleUnload+0x1B974
5> YYToolkit!ModuleUnload+0x1B0AC
6> CallbackManagerMod!_AurieFrameworkInit+0xBF3
7> YYToolkit!ModuleUnload+0x1A6AB
8> YYToolkit!ModuleUnload+0x198A5
9> HoloCure+0x2158963
10> HoloCure+0x22CA8F5
11> HoloCure+0x22B1BFB
12> HoloCure+0x22B1A0D
13> HoloCure+0x229A6CB
14> HoloCure+0x2299FB6
15> HoloCure+0x2208229
16> HoloCure+0x2209B8A
17> HoloCure+0x22363EB
18> HoloCure+0x242542E
19> KERNEL32!BaseThreadInitThunk+0x1D
20> ntdll!RtlUserThreadStart+0x28
############################################################################################
gml_Object_obj_PlayerManager_Draw_64 (line 773)
```

**Comment by PippleCultist** [comment]
The issue with picking up holoboxes crashing should be fixed in v1.1.2. I'm still not sure what you mean by the clients getting abilities part, so please make another issue if you reencounter.
